<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Education</class>
 <widget class="QWidget" name="Education">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1396</width>
    <height>857</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QListWidget" name="educationBanner">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>120</y>
     <width>1121</width>
     <height>81</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="217">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="63">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <item>
    <property name="text">
     <string>
					                		Education 


</string>
    </property>
    <property name="background">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>66</red>
       <green>118</green>
       <blue>253</blue>
      </color>
     </brush>
    </property>
    <property name="foreground">
     <brush brushstyle="DiagCrossPattern">
      <color alpha="255">
       <red>250</red>
       <green>250</green>
       <blue>250</blue>
      </color>
     </brush>
    </property>
   </item>
  </widget>
  <widget class="QListWidget" name="educationBackground_3">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>200</y>
     <width>1121</width>
     <height>71</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="217">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="63">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <item>
    <property name="text">
     <string>
  	      		Sources

</string>
    </property>
    <property name="background">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>169</red>
       <green>169</green>
       <blue>169</blue>
      </color>
     </brush>
    </property>
    <property name="foreground">
     <brush brushstyle="DiagCrossPattern">
      <color alpha="255">
       <red>169</red>
       <green>169</green>
       <blue>169</blue>
      </color>
     </brush>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>30</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>35</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Fish Buddies</string>
   </property>
  </widget>
  <widget class="QLabel" name="fishIcon">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>71</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="Resources.qrc">:/images/fish-logo.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="eduToHome_button_2">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QScrollArea" name="add_a_post">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>350</y>
     <width>411</width>
     <height>281</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>409</width>
      <height>279</height>
     </rect>
    </property>
    <widget class="QLineEdit" name="title_input">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="description_input">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>70</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_general">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>151</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>General Knowledge</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_specific">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>180</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Specific to type</string>
     </property>
    </widget>
    <widget class="QLabel" name="title_label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Title</string>
     </property>
    </widget>
    <widget class="QLabel" name="title_label_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Description</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="source_input">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>120</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="title_label_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>130</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Source</string>
     </property>
    </widget>
    <widget class="QPushButton" name="addPost_button">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>220</y>
       <width>131</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Add a source</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QCheckBox" name="checkBox_generalKnowledge">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>220</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>General Knowledge</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_specificType">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>220</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Specific to type</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>290</y>
     <width>641</width>
     <height>551</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="postLayout"/>
  </widget>
  <widget class="QListWidget" name="eduToHome_button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="217">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="63">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <item>
    <property name="text">
     <string>
            Home
</string>
    </property>
    <property name="background">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>66</red>
       <green>118</green>
       <blue>253</blue>
      </color>
     </brush>
    </property>
    <property name="foreground">
     <brush brushstyle="DiagCrossPattern">
      <color alpha="255">
       <red>250</red>
       <green>250</green>
       <blue>250</blue>
      </color>
     </brush>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="Pref_button">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>40</y>
     <width>131</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Preference Setting</string>
   </property>
  </widget>
  <zorder>educationBackground_3</zorder>
  <zorder>fishIcon</zorder>
  <zorder>add_a_post</zorder>
  <zorder>eduToHome_button_2</zorder>
  <zorder>educationBanner</zorder>
  <zorder>checkBox_generalKnowledge</zorder>
  <zorder>checkBox_specificType</zorder>
  <zorder>verticalLayoutWidget</zorder>
  <zorder>eduToHome_button</zorder>
  <zorder>label</zorder>
  <zorder>Pref_button</zorder>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
